index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Jimmy's Capstone project</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="style.css" />
+  <script
+    src="https://kit.fontawesome.com/1e7e3fccee.js"
+    crossorigin="anonymous"
+  ></script>
+  <script src="script.js"></script>
+  <title>Document</title>
+</head>
+<body>
+  <section class="desktop-head">
+    <div class="top-nav-Desktop">
+      <ul class="top-nav">
+        <li><i class="fa fa-facebook"></i></li>
+        <li><i class="fa fa-twitter"></i></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <nav class="navbar">
+      <div class="menu-container">
+        <a href="#home" class="logo"><img src="./images/iu-logo.png" alt="IU logo" /></a>
+        <div class="menu-nav">
+          <ul>
+            <li class="active-btn"><a href="#" id="home">Home</a></li>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#" class="campaign">Campaign</a></li>
+          </ul>
+        </div>    
+      </div>
+    </nav>
+  </section>
+  
+  
+  <section class="main">
+    <header>
+      <nav class="nav">
+        <div class="top-links">
+          <a class="top-link" href="/about.html">About</a>
+          <a class="top-link" href="#">program</a>
+          <a class="top-link" href="#">Join</a>
+          <a class="top-link" href="#">Sponsor</a>
+          <a class="top-link" href="#">News</a>
+        </div>
+        <div class="hamburger" id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <div class="main-bg absolute"></div>
+    <div class="main-text">
+      <h1>Welcome, English learners.</h1>
+      <h class="green-text">Join the PCC to conquer public speaking forever.</h>
+    </div>
+    <div class="scnd-text">
+      <p>
+        Join over three hundred students to learn and practice presentation skills and public
+        speaking. Compete aganist your colleage and win prestigious prizes.
+      </p>
+    </div>
+    <h2>2024.02.04 ~ 28</h2>
+    <span class="EventLocation">@ ThecEnglish Club, Alexandria, Egypt</span>
+  </section>
+
+  <section class="program-container">
+    <h1>Main Program</h1>
```

Comment:
- [ ] I kindly request that you avoid using more than one `<h1>` tag on each of your pages. This guideline is in place to **ensure consistency and improve the overall structure** and readability of your content. By limiting the usage of `<h1>` tags to one per page, **you provide a clear and hierarchical structure for your headings**, making it easier for users to navigate and comprehend the information on your website.🙏